core/src/main/res/drawable/custom_dialog_shape_selector_left.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please rename the files to something like `filter_dialog_...` instead of `custom_dialog_...`. This makes it easier for other developers to understand what the files are used for.

Reply:
ok